.spectral.yml

```diff
@@ -1,6 +1,5 @@
 extends: [[spectral:oas, all]]
 rules:
-  operation-default-response: off
```

Comment:
curious why this used to be a lint rule?

Reply:
So that linting would pass while we were bootstrapping the spec to this point.

Reply:
Oh; I see. This was turning *off* linting, and now it's back on. I misread it as linting for not having default responses -- which now that I think about it makes a lot less sense.